python/cuml/ensemble/randomforest_common.py

```diff
@@ -0,0 +1,44 @@
+#
+# Copyright (c) 2020, NVIDIA CORPORATION.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+
+def _check_fil_parameter_validity(depth, storage_format, algo):
```

Comment:
because we can not use `fil_sparse_format=True` if the `algo=tree_reorg` or `algo=batch_tree_reorg`. If we set it `fit_sparse_format=true` then it will throw an exception while trying to create the fil format of the forest.